tests/lcs/core/chunks/sort.livecodescript

```diff
@@ -0,0 +1,37 @@
+script "CoreChunksSort"
+/*
+Copyright (C) 2019 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestSortItems
```

Comment:
@livecodepanos Okay so I think you misunderstood me here... What we need to test is the valueref set codepath in MCChunk - this codepath is only used in specific cases, one of them being 'sort'. i.e. We can use `sort button 1` and `sort card 1` to check for the 'invalid container' error (or not as the case may be)

Reply:
Okay so that isn't going to work - sort is very specific - we'll have to ignore this case for now.